tests/mpd/protocol/__init__.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, *args, **kwargs):
         self.response = []
 
     def queue_send(self, data):
+        data = data.decode('utf-8')
```

Comment:
`data` is one small step away from being sent over the network, so it wouldn't be unnatural to keep it as bytes. If we change this to the following, it should work correctly on both Python 2 and 3:

``` python
if isinstance(data, compat.text_type):
    data = data.encode('utf-8')
lines = (line for line in data.split(b'\n') if line)  # Note the explicit 'b' here
...
```

Reply:
OK, sounds sensible. Will update.

Reply:
Hmm but actually, no. This isn't going over the network and don't we want unicode here so that we can just write unicode in the tests?